try:
    file = open('eeee','r')
except Exception as e:
    print(e)